TOC changes from Luis

selhorn opened the pull request:
## Reviewers
@jputrino 

## Fixes 
Adding TOC stickiness, expand/collapse, and style/font changes. The behavior of the top mega menu has also changed slightly, to overlap the TOC.

Commit:
TOC changes from Luis

jputrino commented on f5_sphinx_theme/static/css/f5-theme.css:
```diff
@@ -949,47 +1031,3 @@ table.docutils.footnote, table.docutils.footnote > tbody, table.docutils.footnot
     padding-left: 0;
   }
 }
```
We can't just take out all of the version drop-down elements. This needs to be tested on a doc set that's using the version nav and the elements should be updated as needed to fit with the new styles. 
(f5-rest in GitLab)

selhorn replied:
My fault. CSS we were working with was missing that section for some reason. You can see version selector in action here: http://10.145.104.168:8080/products/iapp/iapp-lx/tmos-14_0/  (It's a bit broken but the top-level page shows what you need to see)

jputrino reviewed (changes requested):
Needs to be tested/updated with a project that uses the version nav dropdown.

Commit:
added back version stuff

jputrino commented:
@nerdoftech The build failure looks like it has something to do with the version selector js. Can you please take a look?

jputrino committed:
Update package.json

Bumping minimum versions of webpack and webpack-cli to fix broken builds related to https://github.com/webpack/webpack/issues/8082

jputrino commented:
@selhorn -- Should be good to go now! Go ahead and merge and then cut a beta release for testing. :D
